### PR TITLE
Features A/B test on firefox/new (Fixes #6830)

### LINKED ISSUE #6830: euexp02 Design Test on /new: More descriptive features/ "More content on /new"
## Description

This project will create a page in which users can see the three feature areas that are currently "below the fold" and test that against the current /new page as the control.  We hypothesize that some people hitting this page are still on the fence and are interested in learning a bit more about Firefox before making a download decision. We believe users will be more likely to download when seeing content more directly, leading to an increase in installs and active days. 

~[Manapage](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=90737686) - euexp03 Design Test on /new: More descriptive features~

### Estimates

| Estimates |  |
| ------------- | ------------- |
| TL | |
| PM | |
| ENG | |
| SEO | |
| ANA | |
| DES | |
| COPY | |

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Test is vetted and approved by Analytics (cohort sizes set for experiment, duration of experiment estimated)
- [ ] This is an example of a success criteria.

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] Depends on #6831

#### Tasks
- [x] Code the test page (dev)
- [x] Determine experiment parameters including target audience, cohort sizes and duration of test
- [x] Set up traffic cop to parameters provided by Analytics (dev)

[DE Copy](https://docs.google.com/document/d/1b--6mC2oQyaDmqvlllQkVust3IxKKdX6bX4TuQt_u9U/edit)
[EN Copy](https://docs.google.com/document/d/1n2ZZmFgy_5yYhxAuc8P36HDtYjFO0Mz8x3E12nmgUuo/edit?usp=sharing)
[Design assets in Abstract](https://share.goabstract.com/35b1fce8-8c72-4185-a215-7de33ac8d2e0)

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds long-form featires description A/B test to `/en-GB/firefox/new/` and `/de/firefox/new/`.
- Removes jQuery code from `scene1_fxa_modal.js`.
- Depends on #6919

## Issue / Bugzilla link
#6830

## Testing
Note: to enter the experiment visitor must be on Windows in a browser with DNT disabled.

Demo URLs:
- https://bedrock-demo-agibson.oregon-b.moz.works/en-GB/firefox/new/
- https://bedrock-demo-agibson.oregon-b.moz.works/de/firefox/new/

Variations:

- https://bedrock-demo-agibson.oregon-b.moz.works/en-GB/firefox/new/?v=a
- https://bedrock-demo-agibson.oregon-b.moz.works/en-GB/firefox/new/?v=b
- https://bedrock-demo-agibson.oregon-b.moz.works/en-GB/firefox/new/?v=c
- https://bedrock-demo-agibson.oregon-b.moz.works/de/firefox/new/?v=a
- https://bedrock-demo-agibson.oregon-b.moz.works/de/firefox/new/?v=b
- https://bedrock-demo-agibson.oregon-b.moz.works/de/firefox/new/?v=c

Things to check:

- [x] Experiment is triggered correctly for each locale.
- [x] Feature clicks are tracked correctly for each variation.
- [x] Correct experiment data is passed to stub attribution for each variation.
- [x] FxA modal still works as expected for signed-out FF users.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/new/scene1.html [hunk 1]
```diff
@@ -167,24 +170,6 @@ <h4 class="fxa-title">{{ _('You’ve already got the Firefox browser. Now get ev
 {% include 'firefox/includes/schemaorg-app.html' %}
 {% endblock %}
 
-{% block stub_attribution %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/new/scene1.html]: Note: this PR can't merge until this prior experiment has been removed.

File: media/js/firefox/new/scene1_fxa_modal.js [hunk 2]
```diff
@@ -8,7 +8,7 @@
     'use strict';
 
     var client = window.Mozilla.Client;
-    var $downloadButton = $('#download-button-desktop-release .download-link[data-download-os="Desktop"]');
+    var $downloadButton = $('.download-button .download-link[data-download-os="Desktop"]');
```
Comment (alexgibson) [inline media/js/firefox/new/scene1_fxa_modal.js]: Note for reviewer: this change makes both the primary and secondary download buttons on `/de/firefox/new/` and `/en-GB/firefox/new/` trigger the FxA modal for signed-out Firefox users.

File: media/js/firefox/new/experiment-features-variation.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var featuresVarA = document.querySelectorAll('.features ul > li');
+    var featuresVarC = document.querySelectorAll('.c-card-feature');
+
+    function trackFeatureClick(e) {
```
Comment (alexgibson) [inline media/js/firefox/new/experiment-features-variation.js]: Note for reviewer: there is an ask in this experiment to track how many clicks each feature gets, despite the features not being interactive in any way. I tried to convey that this is pretty unlikely to get much data as there is nothing to indicate that these things are clickable, but the requesting team is is adamant that they still want this. I also tried to suggest other ways to measure engagement, such as scroll tracking, but couldn't persuade them to change their minds.

**Comment by alexgibson** [comment]
Ok, this is now ready for code review.

**Comment by craigcook** [comment]
I imagine this red flag has been raised before but if not, I'm raising it here: if variant C is a standout winner and we wish to graduate it to all locales, producing and maintaining all those screenshots in a large number of languages will be pure agony. So much so that I wouldn't even consider that design a viable option in the first place, unless it's just a small subset of locales and others get a simplified fallback.

**Comment by alexgibson** [comment]
Yep this has been raised. If this design we’re ever to win we’d have to rethink the images or find an alternative l10n solution. This doesn’t need to block the experiment, which is just to find out if longer feature descriptions are effective.

**Comment by craigcook** [comment]
The low-res images are 480px wide but get blown up to 600px at their largest so they look pretty bad on my non-retina screen. Maybe we're OK with that for a short-lived experiment. Or a quick hack might be to just use the high-res images alone and let low-res screens take the performance hit.

image

**Comment by alexgibson** [comment]
> The low-res images are 480px wide but get blown up to 600px at their largest so they look pretty bad on my non-retina screen. Maybe we're OK with that for a short-lived experiment. Or a quick hack might be to just use the high-res images alone and let low-res screens take the performance hit.

I'd rather not serve everyone on lower-res devices high-res images to save a bit of bluriness. Let me try and find a solution in CSS.

**Comment by alexgibson** [comment]
@craigcook thanks for the review, I pushed a commit with some fixes. I managed to increase the size of the source images a little, and then prevent them from displaying wider than 520px using CSS.

**Comment by alexgibson** [comment]
This is now rebased and ready for final review / merge.

**Pull request merged**